### PR TITLE
Resurect handling of UPPIdentifiers and use new NeoUtils

### PR DESCRIPTION
**Pull request by peteclark-ft**
We had to apply changes for the lifecycle to a previous version of annotations-rw as we couldn't apply those changes on top of the version that had the new UPPIdentifiers because we still haven't loaded all the new UPPIdentifiers. So this PR might be slightly confusing as some functionality has been released but re-applied:

This pull request has several aspects: 
1. Handling of new UPPIdentifiers for Things (unknown concepts)
2. Annotation relationships managed by the annotations-rw (either v1 or v2) now have a new lifecycle property which indicates whether the relationship was created by the annotations-rw or the content-rw (i.e. Brands).  (This is already live)
3. Use the New Neoutils

### CODE REVIEW

File: annotations/cypher_test.go [hunk 1]
```diff
@@ -647,6 +737,51 @@ func cleanUp(t *testing.T, contentUUID string, conceptUUIDs []string) {
 	}
 }
 
+/*func checkDbClean(db *neoism.Database, t *testing.T) {
```
Comment (BerniVarga) [inline annotations/cypher_test.go]: Is this function still needed?

File: annotations/cypher_test.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ const (
 	brandUUID         = "8e21cbd4-e94b-497a-a43b-5b2309badeb3"
 	v2PlatformVersion = "v2"
 	v1PlatformVersion = "v1"
-	contentLifecycle  = "content"
+	contentLifecyle   = "content"
```
Comment (scott-ace-newton) [inline annotations/cypher_test.go]: typo*

**Comment by scott-ace-newton** [comment]
lgtm :+1:

**Comment by scott-ace-newton** [comment]
:+1:

**Pull request merged**